src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -0,0 +1,51 @@
+namespace System
+{
+    internal static class HexConverter
+    {
+        public static char ToCharUpper(int value)
+        {
+            value &= 0xf;
```

Comment:
Nit: I think upper case is common here.

```suggestion
            value &= 0xF;
```
Same below.

Reply:
Changed